Form-Controls/styles.css

```diff
@@ -0,0 +1,81 @@
+body
+{
+background-image:url("./image/bg_pic.jpg") ;
+background-repeat: no-repeat;
+background-size: cover;
+text-align: center;
+align-items: center;
+font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+
+}
+h1{
+    border-radius: 2rem;
+    border-bottom: #e0dada solid 1.5rem;
+    color: rgb(247, 245, 243);
+    font-size: 3.5rem;
+    font-weight: bold;
+}
+
+main{
+    display: flex;
+    flex-direction: row;
+    height: 32rem;
+    border:4rem solid #0f4383; 
+    border-radius: 1rem;      
+    background-color: #e0dada;  
+    background: linear-gradient(45deg, #1a297e, #5b2092, #140fa0, #3a8860); 
+    box-shadow: 0 0.4rem 3rem rgb(253, 250, 250);
+    transition: border-color 0.3s ease, transform 0.3s ease; 
+    justify-content: center;   
+    align-items: center;  
+    text-align: center; 
+    cursor: pointer;   
+    line-height: 1.5rem;    
+    
+}
+main:hover{
```

Comment:
In general, don't write hover styles for non-interactive elements. Hover is an indicator an element is interactive. Here's an example from a common design system

https://m2.material.io/design/interaction/states.html#hover

Reply:
Of course, I completely agree with you. I removed the hover I used on the main style, which was unnecessary. So thank you again for your assistance. I am happy to be commented by you.